apps/src/maze/maze.js

```diff
@@ -76,7 +79,8 @@ module.exports = class Maze {
 
     if (level.map && level.shapeShift) {
       for (let i = 1, max = Math.random() * 4; i < max; i++) {
-        level.map = utils.rotate(level.map);
+        // TODO @snickell ESM - rotate doesn't appear to be present in `../utils`... or anywhere else. This may have already been a broken reference, but improved import syntax discovered it?
+        level.map = rotate(level.map);
```

Comment:
I believe `utils.rotate` was an invalid reference in `staging` that our existing reference checker couldn't catch with old-style imports. I'm not sure what to substitute here, I couldn't infer its function from context. No `function rotate()` is found anywhere in our source. I haven't checked history yet though.